js/main.js

```diff
@@ -82,9 +82,9 @@ function handleLoad(event) {
 }
 
 if (data.view === 'entries') {
-  handleClick();
+  switchView();
 } else if (data.view === 'entry-form') {
-  handleNew();
+  switchView();
 }
```

Comment:
Both blocks of this conditional have the same work. That means no conditional is necessary. just call `switchView()`